evo-delivery-app/evo-delivery-frontend/src/features/map/LeafletMap.tsx

```diff
@@ -17,8 +17,8 @@ export const LeafletMap = ({
   return (
     <MapContainer center={center} zoom={zoom} scrollWheelZoom={scrollWheelZoom} zoomControl={false} {...props}>
       <TileLayer
-        attribution='&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors'
-        url='https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
+        attribution='<a href="https://www.maptiler.com/copyright/" target="_blank">&copy; MapTiler</a> <a href="https://www.openstreetmap.org/copyright" target="_blank">&copy; OpenStreetMap contributors</a>'
```

Comment:
Could you please explain what this change means?

Reply:
see https://cloud.maptiler.com/maps/pastel/I forgot I changed it, we can keep the original if you didn't like